### PR TITLE
Floating Menu: Shapes background color picker

### LINKED ISSUE #10933: UI Simplification: Floating menu: Shape Menus Color Picker
## Feature Description

According to figma, shapes should have a color picker in their floating menus. 

There's 2 color pickers shown in figma, which makes me think it's not current so let's just stick with the condensed one that's getting used by text for consistency and keep it next to the eyedropper not the border input.

Screen Shot 2022-03-15 at 4 18 55 PM

## Acceptance Criteria

- Both rectangle and round shape menu variants have a color picker.
- Relevant tests added 
- Works with keyboard
- Tooltip that says "Open color picker" (just like text)

### LINKED ISSUE #10934: UI Simplification: Floating menu: Shape Menus Opacity
## Feature Description

According to figma, shapes should have an opacity input in their floating menus. 
Should be the same opacity input as images and videos and stickers.

Screen Shot 2022-03-15 at 4 18 55 PM

## Acceptance Criteria

- Both rectangle and round shape menu variants have an opacity input.
- Relevant tests added 
- Works with keyboard

### PR DESCRIPTION
**Pull request by BrookeGraham**
## Context

The floating menu for Shapes still had placeholder button for the color picker, eyedropper and opacity. We are just added that here. 

## Summary
This pr is blocked (branched off of) by `sidebar`. We need the update to the color popup in order for the positioning to be correct. 

## Relevant Technical Choices
Looks like the shared `color` component for all the floating menus has a `width` prop that was being sent to `Color`, but it was never being used. I hooked that up so that the width of the inputs can be set dynamically based on those inputs. 

## To-do
n/a

## User-facing changes
Screen Shot 2022-03-16 at 12 38 30 PM

## Testing Instructions

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Open floating menu for a shape element
2. check that changing color and opacity works as expected and is synced with design panel. 

### Does this PR have a security-related impact?
no

### Does this PR change what data or activity we track or use?
I added tracking (to be in sync with other floating menu tracking) 

### Does this PR have a legal-related impact?
no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/googleforcreators/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This code is covered by automated tests (unit, integration, and/or e2e) to verify it works as intended ([docs](https://github.com/googleforcreators/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #10933 
Fixes: #10934

### CODE REVIEW

File: packages/story-editor/src/components/form/color/color.js [hunk 1]
```diff
@@ -100,6 +101,7 @@ const Color = forwardRef(function Color(
     pickerPlacement = PLACEMENT.RIGHT_START,
     isInDesignMenu = false,
     hasInputs = true,
+    width,
```
Comment (BrookeGraham) [inline packages/story-editor/src/components/form/color/color.js:104]: This was being sent in by the [shared color component](https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/components/floatingMenu/elements/shared/color.js#L34-L56) inside floating menus, but wasn't ever hooked up. This lead us to get the wrong width in floating menus.

File: packages/story-editor/src/components/floatingMenu/elements/shapeColor.js [hunk 2]
```diff
@@ -17,20 +17,48 @@
...
+      backgroundColor: state.selectedElements[0].backgroundColor,
+      updateSelectedElements: actions.updateSelectedElements,
+    })
+  );
+  const pushUpdate = useCallback(
+    (value) => {
+      trackEvent('floating_menu', {
+        name: 'set_shape_color',
+        element: 'shape',
+      });
```
Comment (BrookeGraham) [inline packages/story-editor/src/components/floatingMenu/elements/shapeColor.js:42]: Saw that we have some tracking in floating menus, added this here. Not sure if we want to this or not.

Comment (BrittanyIRL) [reply packages/story-editor/src/components/floatingMenu/elements/shapeColor.js:42]: Miina made a ticket to fill in the tracking so you adding it here is great, saves us the effort on this action later. https://github.com/GoogleForCreators/web-stories-wp/issues/10871

**Review by BrittanyIRL — APPROVED** [review]
🟨 🟣 💛 🟢 💙 🟥  
(shapes)

Works great, thanks for such a fast turn around!

**Review by samwhale — APPROVED** [review]
✨

**Review by miina — APPROVED** [review]
Looks great! 🎉

**Pull request merged**